Week-14/Binary_Search_2/Search_a_2D_Matrix

```diff
@@ -0,0 +1,46 @@
+function findTarget(row, column, arr, target) {
```

Comment:
rename this file to have a .js extension

Reply:
Done